lib/jopenssl23/openssl/ssl.rb

```diff
@@ -214,7 +214,7 @@ def ssl_version=(meth)
           raise ArgumentError, "unknown SSL method `%s'" % meth
         set_minmax_proto_version(version, version)
         @min_proto_version = @max_proto_version = version
-      end unless method_defined? :ssl_version=
```

Comment:
let's keep this as is as it seems unrelated to the PR changes?

Reply:
I believe I had a reason to remove this. I compared this with CRuby version: https://github.com/ruby/ruby/blob/ruby_2_6/ext/openssl/lib/openssl/ssl.rb#L213
which does not have `unless method_defined? :ssl_version=`

I'll double check it and revert if it's fine.

Reply:
Ok, removed.
Only one smalll difference, unrealistic though, (was successful before, now fails with `no protocols available`):

```
ctx.ssl_version = "TLSv1"
ctx.min_version = OpenSSL::SSL::TLS1_VERSION
ctx.max_version = OpenSSL::SSL::TLS1_2_VERSION
```

Reply:
Thanks, will need to :eyes: at this since it changed behavior - which I did not expect to happen.